src/datastore_export/load_datastore_backups_to_big_query_service.py

```diff
@@ -30,15 +30,15 @@ def load(self, source_uri, kinds):
             DATASET_ID, self.location
         )
 
-        load_job_ids = []
+        load_jobs_tuples = []
```

Comment:
Don't use terms like 'tuple, list, array'. You should only care that it's some kind of collection, e.g. load_jobs.

Reply:
done.